app/src/main/java/org/mozilla/fenix/experiments/NimbusFeatures.kt

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.experiments
+
+import android.content.Context
+import org.mozilla.experiments.nimbus.mapKeysAsEnums
+import org.mozilla.fenix.Config
+import org.mozilla.fenix.FeatureFlags
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.ext.getVariables
+
+/**
+ * Component for exposing nimbus Feature Variables.
+ * For more information see https://experimenter.info/feature-variables-and-me
+ *
+ * @param context - A [Context] for accessing the feature variables from nimbus.
+ */
+class NimbusFeatures(private val context: Context) {
+
+    val homeScreen: HomeScreenFeatures by lazy {
+        HomeScreenFeatures(context)
+    }
+
+    /**
+     * Component that indicates which features should be active on the home screen.
+     */
+    class HomeScreenFeatures(private val context: Context) {
+        /**
+         * `FeatureId.HOME_PAGE` feature; the complete JSON, is shown here:
+         *
+         * ```json
+         * {
+         *     "sections-enabled": {
+         *         "topSites": true,
+         *         "recentlySaved": false,
+         *         "jumpBackIn": false,
+         *         "pocket": false,
+         *         "recentExplorations": false
+         *     }
+         * }
+         * ```
+         */
+
+        /**
+         * This enum accompanies the `FeatureId.HOME_PAGE` feature.
+         *
+         * These names here should match the names of entries in the JSON.
+         */
+        @Suppress("EnumNaming")
+        private enum class HomeScreenSection(val default: Boolean) {
+            topSites(true),
+            recentlySaved(false),
+            jumpBackIn(false),
+            pocket(false),
+            recentExplorations(false);
+
+            companion object {
+                /**
+                 * CreateS a map with the corresponding default values for each sections.
+                 */
+                fun toMap(context: Context): Map<HomeScreenSection, Boolean> {
+                    return values().associate { section ->
+                        val channelDefault = if (section == pocket) {
+                            FeatureFlags.isPocketRecommendationsFeatureEnabled(context)
```

Comment:
We need it here because we are prioritizing the `HomeScreenSection.default` and then falling back to the feature availability for compatibility with the actual behavior on nightly. At the moment, **pocket is not enabled to the whole nightly population** [see](https://github.com/mozilla-mobile/fenix/blob/903c25095a3d5ec710ea57c88ffa3623355cdf38/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt#L85) only a segment of it. If we rely only on `HomeScreenSection.default || Config.channel == ReleaseChannel.Nightly` we will be activating it to the whole nightly population that wasn't the product intention.